src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -345,8 +345,8 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     /**
      * Given an existing path, convert it to a path relative to a given starting path.
      *
-     * @param string $endPath   Absolute path of target
-     * @param string $startPath Absolute path where traversal begins
+     * @param string $endPath   Path of target
+     * @param string $startPath Path where traversal begins
```

Comment:
I didn't see that test case initially. I'm this case, we IMO should indeed support this and update the docblock as suggested. But I think we should indeed think about deprecating this behaviour.